src/lib.rs

```diff
@@ -82,6 +83,10 @@ mod macos;
 #[cfg_attr(docsrs, doc(cfg(target_os = "windows")))]
 mod windows;
 
+#[cfg(feature = "windows-native")]
+#[cfg_attr(docsrs, doc(cfg(fall(feature = "linux-native", target_os = "linux"))))]
```

Comment:
not sure if "fall" is valid.
Also probably doc  should only be generated for target_os windows and feature windows-native.

Reply:
Yeah, I kinda missed this when I copy and pasted my "template" together